app/models/mixins/per_ems_worker_mixin.rb

```diff
@@ -97,7 +97,7 @@ def queue_name_for_ems(ems)
       return "generic" if ems.kind_of?(Host) && ems.acts_as_ems?
```

Comment:
I think we could do that but it would mean we should remove the `return ems unless ems.kind_of?(ExtManagementSystem)` below too. Im not sure which (if any) other entities use this code.